runtime/src/main/java/org/corfudb/runtime/collections/StreamManager.java

```diff
@@ -272,7 +273,11 @@ public void run() {
                                     ).collect(Collectors.toList())));
 
                     if (!entries.isEmpty()) {
-                        CorfuStreamEntries callbackResult = new CorfuStreamEntries(entries);
+                        Timestamp timestamp = Timestamp.newBuilder()
+                                .setSequence(logData.getGlobalAddress())
+                                .setEpoch(epoch)
```

Comment:
is it required to expose the epoch too to the client?  Isnt globalAddress enough?